packages/clay-dropdown/src/ClayActionsDropdown.soy

```diff
@@ -8,6 +8,7 @@
 	{@param spritemap: string}
 	{@param? _handleButtonClick: any}
 	{@param? _handleItemClick: any}
+	{@param? ariaLabel: string}
```

Comment:
hey @boton, can you declare this parameter in the state of `ClayActionsDropdown`?